updateUi.py

```diff
@@ -0,0 +1,92 @@
+from tkinter import *
+from GameData import GameData
+from State import State
+from StatEnum import Stat
+from CharacterInstance import CharacterInstance
+from UserLogs import UserLogs
+
+
+class updateUi(Frame):
+    def __init__(self, user_logs, master=None):
```

Comment:
"""
inits a new UpdateUI object to handle updating of a CharacterInstance with a new state
"""